crates/ruff_python_semantic/src/model.rs

```diff
@@ -2199,36 +2159,11 @@ bitflags! {
         /// class Baz(Foo, Bar):
         ///     pass
         /// ```
-        const CLASS_BASE = 1 << 24;
+        const CLASS_BASE = 1 << 23;
 
         /// The model is visiting a class base that was initially deferred
         /// while traversing the AST. (This only happens in stub files.)
-        const DEFERRED_CLASS_BASE = 1 << 25;
-
-        /// The model is in an attribute docstring.
-        ///
-        /// An attribute docstring is a string literal immediately following an assignment or an
-        /// annotated assignment statement. The context in which this is valid are:
-        /// 1. At the top level of a module
-        /// 2. At the top level of a class definition i.e., a class attribute
-        ///
-        /// For example:
-        /// ```python
-        /// a = 1
-        /// """This is an attribute docstring for `a` variable"""
-        ///
-        ///
-        /// class Foo:
-        ///     b = 1
-        ///     """This is an attribute docstring for `Foo.b` class variable"""
-        /// ```
-        ///
-        /// Unlike other kinds of docstrings as described in [PEP 257], attribute docstrings are
-        /// discarded at runtime. However, they are used by some documentation renderers and
-        /// static-analysis tools.
-        ///
-        /// [PEP 257]: https://peps.python.org/pep-0257/#what-is-a-docstring
-        const ATTRIBUTE_DOCSTRING = 1 << 26;
```

Comment:
I think we should leave this, actually. It was added in https://github.com/astral-sh/ruff/pull/11315 though I don't think it was "finished".

Reply:
Oh, good point -- https://github.com/astral-sh/ruff/issues/10347 is still open

Reply:
added it back